int64                    frame_id
geometry_msgs/Transform  T_c_w
int32                    lm_count
std_msgs/Int64MultiArray lm_id_data
geometry_msgs/Vector3[]  lm_3d_data
int32                    lm_outlier_count
std_msgs/Int64MultiArray lm_outlier_id_data
